### ISSUE TITLE
SC 1.3.4 - Proposal for change of wording

### ISSUE THREAD

**Issue by MakotoUeki** [w3c/wcag21#665]
-- Current wording --

Success Criterion 1.3.4 Identify Common Purpose

In content implemented using markup languages, for each user interface component that serves a purpose identified in the Common Purposes for User Interface Components section, that purpose can be programmatically determined.

--

I find this type of sentence structure – in which the subject and verb do not appear till later – hard to follow and translate. 

-- Proposal --

Success Criterion 1.3.4 Identify Common Purpose

The common purpose of each user interface component (identified the Common Purposes for User Interface Components section) can be programmatically determined when content is implemented using markup languages.

--

**Comment by awkawk** [comment]
(Official WG response)
The wording has been significantly updated, we think this addresses the issue.
